hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts

```diff
@@ -298,6 +305,16 @@ class HassioSnapshotDialog extends LitElement {
   }
 
   private async _partialRestoreClicked() {
+    if (
+      this.supervisor !== undefined &&
+      this.supervisor.info.state !== "running"
+    ) {
+      await showAlertDialog(this, {
+        title: "Could not restore snapshot",
+        text: `Restoring snapshots is disabled. System is not running (${this.supervisor.info.state})`,
```

Comment:
Maybe change wording to something like:

Restoring a snapshot is not possible right now because the system is in ${this.supervisor.info.state} state.